warehouse/query-core/src/main/java/datawave/query/iterator/NestedQueryIterator.java

```diff
@@ -81,23 +83,27 @@ public T next() {
     @Override
     public void remove() {
         currentNest.remove();
-
     }
 
     @Override
     public void initialize() {
         if (null == currentNest) {
             popNextNest();
-        } else
+        } else {
             currentNest.initialize();
-
+        }
     }
 
     @Override
     public T move(T minimum) {
         return currentNest.move(minimum);
     }
 
+    @Override
+    public void seek(Range range, Collection<ByteSequence> columnFamilies, boolean inclusive) throws IOException {
+        currentNest.seek(range, columnFamilies, inclusive);
```

Comment:
might this need to go beyond currentNest if currentNest is exhausted but there are more keys?